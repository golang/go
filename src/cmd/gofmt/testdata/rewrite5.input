//gofmt -r=x+x->2*x

// Copyright 2011 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

// Rewriting of expressions containing nodes with associated comments to
// expressions without those nodes must also eliminate the associated
// comments.

package p

func f(x int) int {
	_ = x + x // this comment remains in the rewrite
	_ = x /* this comment must not be in the rewrite */ + x
	return x /* this comment must not be in the rewrite */ + x
}
