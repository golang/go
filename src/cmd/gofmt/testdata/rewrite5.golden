//gofmt -r=x+x->2*x

// Copyright 2011 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

// Rewriting of expressions containing nodes with associated comments to
// expressions without those nodes must also eliminate the associated
// comments.

package p

func f(x int) int {
	_ = 2 * x // this comment remains in the rewrite
	_ = 2 * x
	return 2 * x
}
